### DISCUSSION TITLE
模型路径是对的，提示说找不到模型

### DISCUSSION THREAD

**Discussion by nissansz** [PaddlePaddle/PaddleOCR#14177] [Q&A]
### 🔎 Search before asking

- [X] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.
- [X] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.
- [X] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### 🐛 Bug (问题描述)

L:\paddle\PaddleOCR-2.9.1>python312 tools/infer/predict_rec.py --image_dir='L:\paddle\PaddleOCR-2.9.1\doc\imgs_words_en/word_10.png' --rec_model_dir='L:/paddle/models/ch_PP-OCRv4_rec_server_infer/'
Traceback (most recent call last):
  File "L:\paddle\PaddleOCR-2.9.1\tools\infer\predict_rec.py", line 877, in <module>
    main(utility.parse_args())
  File "L:\paddle\PaddleOCR-2.9.1\tools\infer\predict_rec.py", line 839, in main
    text_recognizer = TextRecognizer(args)
                      ^^^^^^^^^^^^^^^^^^^^
  File "L:\paddle\PaddleOCR-2.9.1\tools\infer\predict_rec.py", line 154, in __init__
    ) = utility.create_predictor(args, "rec", logger)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "L:\paddle\PaddleOCR-2.9.1\tools\infer\utility.py", line 220, in create_predictor
    raise ValueError(
ValueError: not find model.pdmodel or inference.pdmodel in 'L:/paddle/models/ch_PP-OCRv4_rec_server_infer/'

### 🏃‍♂️ Environment (运行环境)

win10 python 312

### 🌰 Minimal Reproducible Example (最小可复现问题的Demo)

L:\paddle\PaddleOCR-2.9.1>python312 tools/infer/predict_rec.py --image_dir='L:\paddle\PaddleOCR-2.9.1\doc\imgs_words_en/word_10.png' --rec_model_dir='L:/paddle/models/ch_PP-OCRv4_rec_server_infer/'
Traceback (most recent call last):
  File "L:\paddle\PaddleOCR-2.9.1\tools\infer\predict_rec.py", line 877, in <module>
    main(utility.parse_args())
  File "L:\paddle\PaddleOCR-2.9.1\tools\infer\predict_rec.py", line 839, in main
    text_recognizer = TextRecognizer(args)
                      ^^^^^^^^^^^^^^^^^^^^
  File "L:\paddle\PaddleOCR-2.9.1\tools\infer\predict_rec.py", line 154, in __init__
    ) = utility.create_predictor(args, "rec", logger)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "L:\paddle\PaddleOCR-2.9.1\tools\infer\utility.py", line 220, in create_predictor
    raise ValueError(
ValueError: not find model.pdmodel or inference.pdmodel in 'L:/paddle/models/ch_PP-OCRv4_rec_server_infer/'

**Comment by Cupcc** [comment]
截图看下 L:/paddle/models/ch_PP-OCRv4_rec_server_infer/ 真的有吗？

**Comment by nissansz** [comment]
![image](https://github.com/user-attachments/assets/165fc443-ecf6-4d85-9caf-0e1afaa282b9)

**Comment by jingsongliujing** [comment]
![image](https://github.com/user-attachments/assets/e67c8230-7454-4d7f-ad0c-33cacb496adb)
把最后一个斜杠去掉试试

**Comment by nissansz** [comment]
是不是默认加载了什么模型？不管改什么版本模型都是一样的结果，准确率也是一模一样

python tools/infer/predict_rec.py --image_dir='L:\paddle\PaddleOCR-2.9.1\doc\imgs_words\ch\word_4.jpg' --rec_model_dir='L:\paddle\models\ch_PP-OCRv4_rec_server_infer'
